fxevent/zap_test.go

```diff
@@ -131,64 +131,73 @@ func TestZapLogger(t *testing.T) {
 			give:        &Supplied{TypeName: "*bytes.Buffer"},
 			wantMessage: "supplied",
 			wantFields: map[string]interface{}{
-				"type": "*bytes.Buffer",
+				"type":   "*bytes.Buffer",
+				"module": "",
 			},
 		},
 		{
 			name:        "SuppliedError",
 			give:        &Supplied{TypeName: "*bytes.Buffer", Err: someError},
 			wantMessage: "supplied",
 			wantFields: map[string]interface{}{
-				"type":  "*bytes.Buffer",
-				"error": "some error",
+				"type":   "*bytes.Buffer",
+				"module": "",
+				"error":  "some error",
 			},
 		},
 		{
 			name: "Provide",
 			give: &Provided{
 				ConstructorName: "bytes.NewBuffer()",
+				ModuleName:      "myModule",
 				OutputTypeNames: []string{"*bytes.Buffer"},
 			},
 			wantMessage: "provided",
 			wantFields: map[string]interface{}{
 				"constructor": "bytes.NewBuffer()",
 				"type":        "*bytes.Buffer",
+				"module":      "myModule",
 			},
 		},
 		{
 			name:        "Provide with Error",
 			give:        &Provided{Err: someError},
 			wantMessage: "error encountered while applying options",
 			wantFields: map[string]interface{}{
-				"error": "some error",
+				"error":  "some error",
+				"module": "",
```

Comment:
same. maybe we should omit this field if it's empty.

on the zap end, you can do this easily like so:

```
func moduleField(name string) zap.Field {
  if len(name) == 0 { return zap.Skip() }
  return zap.String("module", name)
}
```